virttest/utils_selinux.py

```diff
@@ -278,6 +278,26 @@ def set_context_of_file(filename, context, selinux_force=False):
     logging.debug("Set context of %s success.", filename)
 
 
+def check_context_of_file(filename, label, selinux_force=False):
```

Comment:
@bssrikanth Do you need to set the default value for label or that will have to be taken care in calling function?

Reply:
it has to be taken care by calling function.